images/alevinfry/Dockerfile

```diff
@@ -5,6 +5,6 @@ LABEL maintainer="ccdl@alexslemonade.org"
 RUN apt-get update && apt-get install -y procps
 
 # Install alevin-fry using bioconda
-RUN conda install -c bioconda alevin-fry 
+RUN conda install -c conda-forge -c bioconda alevin-fry=0.2.0
```

Comment:
> In that update, there is a dependency on the most recent version of libcxx, requiring the install using the -conda-forge channel.

I think this is fine, but I did wonder whether this could/should be installed with apt. As I write that, I actually think it may make more sense to install dependencies like this with `conda` (I assume there are other dependencies that are coming in here as well),  but we will probably want to make an accounting of what those all of those dependencies are at some point to keep things as stable/reproducible as possible. I don't think we are there yet, but we may want to do this when we finalize the package set we are using.